clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -14751,9 +14751,58 @@ void AsmRule::registerMatcher(ast_matchers::MatchFinder &MF) {
       this);
 }
 
+std::string GetAsmLop3Expr(const std::string &TemplateString,
```

Comment:
the function name convention is using lower case for first letter

Reply:
fixed